web/src/components/organisms/User/index.tsx

```diff
@@ -16,8 +16,19 @@ export type MyDataType = {
 };
 
 const User: React.FC<Props> = ({ onPluginSelect }) => {
-  const { myData, plugins } = useHooks();
-  return <UserPage myData={myData} plugins={plugins} onPluginSelect={onPluginSelect} />;
+  const pageSize = 40;
```

Comment:
Move this in to the hooks.ts file and keep it out of the function's scope (it never changes) and then pass it with the other values in the return of the hooks function.